### PR TITLE
site: Add netlify.toml

### PR DESCRIPTION
**Pull request by ahmetb**
From: https://github.com/kubernetes/community/blob/master/github-management/subproject-site-requests.md#example-netlify-configuration

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+publish = "site/public/"
+command = "hugo --gc"
+
+[build.environment]
+HUGO_VERSION = "0.65.1"
+
+[context.production.environment]
+HUGO_ENV = "production"
+HUGO_ENABLEGITINFO = "true"
+
```
Comment (corneliusweig) [inline netlify.toml]: Wouldn't it make sense to add this already?
```
HUGO_BASEURL = "https://krew.sigs.k8s.io/"
```

File: netlify.toml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+base = "site/"
+publish = "site/public/"
+command = "hugo --gc"
+
+[build.environment]
+HUGO_VERSION = "0.65.1"
+
+[context.production.environment]
+HUGO_ENV = "production"
+HUGO_ENABLEGITINFO = "true"
```
Comment (corneliusweig) [inline netlify.toml]: I suspect that we don't need both the env var and the flag? Let's choose one.

File: netlify.toml (hunk 1 above)
Comment (ahmetb) [reply netlify.toml]: it's already in config.yaml, so I don't think it's needed.

File: netlify.toml (hunk 2 above)
Comment (ahmetb) [reply netlify.toml]: u r right.

File: netlify.toml (hunk 1 above)
Comment (corneliusweig) [reply netlify.toml]: I see! makes sense

**Pull request merged**